tests/test_query.py

```diff
@@ -160,3 +159,46 @@ def test_complex_query(self):
 
         self.assertEqual(timestamps, ["20170101T0000-20190131T235959"])
         self.assertEqual(data, [8.519842147827148])
+
+    def test_run_job_on_queue_directly(self):
```

Comment:
Do we still need this test? It is fetching quite a lot of data and the initial issue is circumvented differently